api/src/main/java/ca/bc/gov/educ/studentdatacollection/api/service/v1/ValidationRulesService.java

```diff
@@ -103,14 +103,16 @@ public void updatePenMatchAndGradStatusColumns(SdcSchoolCollectionStudentEntity
         log.error("PenMatchRecord in priority queue is empty for matched status, this should not have happened.");
         throw new StudentDataCollectionAPIRuntimeException("PenMatchRecord in priority queue is empty for matched status, this should not have happened.");
       }
+    }else if (StringUtils.isNotEmpty(penMatchResultCode) && multiPenMatchResults.contains(penMatchResultCode)) {
+      student.setPenMatchResult("MULTI");
     }else{
       student.setPenMatchResult("NEW");
     }
 
-    if(student.getAssignedStudentId() != null) {
-        final var gradResult = this.restUtils.getGradStatusResult(UUID.randomUUID(), SdcSchoolCollectionStudentMapper.mapper.toSdcSchoolStudent(student));
-        log.info("Grad status for SDC student {} :: is {}", student.getSdcSchoolCollectionStudentID(), gradResult);
-    }
+//    if(student.getAssignedStudentId() != null) {
+//        final var gradResult = this.restUtils.getGradStatusResult(UUID.randomUUID(), SdcSchoolCollectionStudentMapper.mapper.toSdcSchoolStudent(student));
+//        log.info("Grad status for SDC student {} :: is {}", student.getSdcSchoolCollectionStudentID(), gradResult);
+//    }
     student.setIsGraduated(false);
```

Comment:
Is this commented out code needed?

Reply:
Needed. Waiting on Grad Status check on the Grad team side.